packages/teams-js/src/public/dialog.ts

```diff
@@ -16,21 +17,44 @@ import { runtime } from './runtime';
  *
  * @beta
  */
+export interface SdkResponse {
+  err: string;
+  result: string | object;
+}
+
 export namespace dialog {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  export type PostMessageChannel = (message: any, onComplete?: (status: boolean, reason?: string) => void) => void;
+  export type DialogSubmitHandler = (result: SdkResponse) => void;
   /**
    * Allows an app to open the dialog module.
    *
-   * @param dialogInfo - An object containing the parameters of the dialog module
+   * @param urlDialogInfo - An object containing the parameters of the dialog module
    * @param submitHandler - Handler to call when the task module is completed
+   * @param messageFromChildHandler - Handler that triggers if dialog tries to send a message to the app.
+   *
+   * @returns a Handler that is triggerd to send a message to the dialog.
```

Comment:
Maybe this comment should be "@returns a function that can be used to send messages to the dialog"

Reply:
Yes, makes more sense!